users/untrusted.rst

```diff
@@ -5,14 +5,14 @@ Untrusted (Encrypted) Devices
 =============================
 
 It is possible to set a password on a folder when it's shared with another
-device. Data sent will be encrypted by this password, and data received will
-be decrypted by the same password.
+device. Data sent will be encrypted at rest by this password, and data
+received will be decrypted by the same password.
```

Comment:
I'd recommend rewriting the entire sentence to make it clearer. Now it doesn't make sense at all anymore, even though I of course know what it means, but the point here is to be more clear to people who aren't familiar with this: Data cannot both be "sent"/"received" and "at rest".